src/definitions/modules/dropdown.js

```diff
@@ -412,7 +412,9 @@ $.fn.dropdown = function(parameters) {
             module.refresh();
           },
           menu: function(values) {
-            $menu.html( templates.menu(values, fields,settings.preserveHTML,settings.className));
+            const response = values;
+            response[fields.values] = values.values;
```

Comment:
Why the separate constant?

What about doing this instead: 
```javascript
if(!values[fields.values] && values.values) values[fields.values] = values.values;
```


TBH, the previous solution was more logical to me (yes, i understand you would have to adjust `get.selectValues` as well at many places)

Reply:
Ok, I can see the initial solution might have been better.
Added commits to build up on that initial solution.
Also added commit to extract local variable in `get.selectValues`, but this is probably a matter of taste

Reply:
Thanks, much better approach!